packages/plugin/webpack/src/util/AssetRelocatorPatch.ts

```diff
@@ -32,20 +35,29 @@ export default class AssetRelocatorPatch {
                   throw new Error('The installed version of @vercel/webpack-asset-relocator-loader does not appear to be compatible with Forge');
                 }
 
+                if (this.isProd) {
+                  return originalInjectCode.replace(
+                    '__dirname',
+                    this.nodeIntegration
+                      // In production the assets are found one directory up from
+                      // __dirname
+                      //
+                      // __dirname cannot be used directly until this PR lands
+                      // https://github.com/jantimon/html-webpack-plugin/pull/1650
+                      ? 'require("path").resolve(require("path").dirname(__filename), "..")'
+                      // If nodeIntegration is disabled, we replace __dirname
+                      // with an empty string so no error is thrown at runtime
+                      : '""',
```

Comment:
Would it be possible to extract this into its own function? I'm not really a fan of multi-line ternary statements.

Reply:
Seems like it. I tested locally and committed in https://github.com/electron-userland/electron-forge/pull/2396/commits/2daf05377bf03b0b981cd58e7767ac94c484773f